_targets.R

```diff
@@ -12,39 +12,38 @@ p1_targets_list <- list(
   tar_target(site_data_01436690, download_nwis_site_data('01436690')),
   tar_target(site_data_01466500, download_nwis_site_data('01466500')),
   
-  tar_target(site_data_csv,
+  tar_target(site_data,
              {
-               out_file <- "1_fetch/out/site_data.csv"
-               list(site_data_01427207, site_data_01432160, 
+               site_data<-list(site_data_01427207, site_data_01432160, 
                     site_data_01436690, site_data_01466500) %>% 
-                 bind_rows() %>% 
-                 write_csv(out_file)
-               return(out_file)
-             },
-             format = "file"
+                 bind_rows() 
+               return(site_data)
+             }
   ),
   
   tar_target(
-    site_info_csv,
-    nwis_site_info(filein = site_data_csv, fileout = "1_fetch/out/site_info.csv")
+    site_info,
+    nwis_site_info(site_data = site_data)
   )
 )
 
 p2_targets_list <- list(
   tar_target(
-    site_data_clean, 
-    process_data(site_data_csv)
+    nwis_data_clean_csv, 
+    process_data(site_data = site_data, 
+                 fileout = "2_process/out/nwis_data_clean.csv")
   ),
   tar_target(
-    cleaned_data,
-    annotate_and_style_data(site_data_clean, site_filename = site_info_csv)
+    cleaned_data_csv,
+    annotate_and_style_data(filein = nwis_data_clean_csv, site_info = site_info,
+                            fileout = "2_process/out/cleaned_data.csv")
```

Comment:
Just need to add `format = "file"` here so that this target will be tracked by `targets` as a file.